docker/test/integration/minifi/core/DockerTestDirectoryBindings.py

```diff
@@ -77,6 +77,7 @@ def put_file_contents(file_abs_path, contents):
         logging.info('Writing %d bytes of content to file: %s', len(contents), file_abs_path)
         with open(file_abs_path, 'ab') as test_input_file:
             test_input_file.write(contents)
+        os.chmod(file_abs_path, 0o0777)
```

Comment:
OoO I did not know this alternate syntax, it's interesting.  But I think `0777` is more standard and more readable.

Reply:
In python that syntax cannot be used.
```
    a = 0777
           ^
SyntaxError: leading zeros in decimal integer literals are not permitted; use an 0o prefix for octal integers
```

Reply:
Yeah, it looks like the 0777 syntax was valid in Python 2 but it got removed in Python 3.  Maybe someone pointed out to them that having both `0123` and `0o123` conflicts with "There's Only One Way To Do It" -- and then they picked the wrong one to keep. :D